sycl/test-e2e/format.py

```diff
@@ -201,6 +201,9 @@ def get_extra_env(sycl_devices):
             if "level_zero:gpu" in sycl_devices and litConfig.params.get("ur_l0_debug"):
                 extra_env.append("UR_L0_DEBUG={}".format(test.config.ur_l0_debug))
 
+            if "level_zero:gpu" in sycl_devices:
+                extra_env.append("UR_ENABLE_LAYERS=UR_LAYER_PARAMETER_VALIDATION")
+
```

Comment:
Do we really need it? We had internal e2e tests running with UR verification/L0 plugin leaks detection enabled (before PI removal) and the only single issue caught by UR verification level was the verification code itself.

Reply:
The plugins used to have some level of parameter checking in the entry points themselves, that's not the case with UR where this only occurs on the validation layer. The failing tests in this PR appear to depend on parameter checking for correctness.

Reply:
1) I'm still not convinced.
2) > The failing tests in this PR appear to depend on parameter checking for correctness.

can you elaborate on this?

Reply:
Hello @aelovikov-intel ,

So, I want to understand your concern. My primary concern is to disable the validation layer by default because this is impacting performance on every call to the UR. Is your concern the fact that we would enable the validation layer during E2E testing?

Why would that be an issue? Don't we want to have more checks during the E2E testing?

Reply:
> Why would that be an issue? 

because it affects timing and could possible hide issues that our customers would see without that validation. Also, how many issues that actually catches?